README.md

```diff
@@ -159,13 +159,13 @@ You can generate a waveform of an audio file using the `FFMpeg\Media\Audio::wave
 method.
 
 This code returns a `FFMpeg\Media\Waveform` instance.
-You can optionally pass dimensions as arguments, see dedicated
+You can optionally pass dimensions as the first two arguments and an array of hex string colors for ffmpeg to use for the waveform, see dedicated
 documentation below for more information.
 
 The ouput file MUST use the PNG extension.
 
 ```php
-$waveform = $audio->waveform(640, 120, ['#00FF00']);
+$waveform = $audio->waveform(640, 120, ['#00FF00/']);
```

Comment:
is that slash placed by purpose?

Reply:
Not that I know of.. :D I've asked the cat, but she denies all allegations.